src/components/headerBody/HeaderBody.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect } from 'react';
+import './headerbody.css';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+// Import Swiper styles
+import 'swiper/css';
+import 'swiper/css/pagination';
+import 'swiper/css/navigation';
+
+// import "./styles.css";
+
+// import required modules
+import { Pagination, Navigation } from 'swiper';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
- [ ] Please, remove the old pieces of commented code present in this file, in order to improve the cleanliness of the repo 💪